src/cui.cpp

```diff
@@ -199,6 +200,11 @@ void Line::log() {
             << "\t" << recv_value << std::endl;
 }
 
+void Line::log_json() {
+  printf("{\"name\":\"%s\",\"PID\":%d,\"UID\":%d,\"RX\":%f,\"TX\":%f }",
+        m_name, m_pid, m_uid, recv_value, sent_value);
+}
```

Comment:
I think a timestamp would be useful (indeed for after-the-fact processing). Adding it to the normal `-t` output might break existing scripts consuming that output, so I'd probably prefer to leave that alone.